From dacfe2fc7b41b800f4f379ce4b1722462b97cab2 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sun, 6 Dec 2020 13:47:18 +0100
Subject: [PATCH 7/9] chtnau8824: Boost analog mic volumes a bit

Both Mic1 (internal analog mic) and Mic2 (headset mic) are a bit soft
with the current settings, boost their volumes a bit.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/codecs/nau8824/EnableSeq.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ucm2/codecs/nau8824/EnableSeq.conf b/ucm2/codecs/nau8824/EnableSeq.conf
index 433e8e283d4b..3650d06485b4 100644
--- a/ucm2/codecs/nau8824/EnableSeq.conf
+++ b/ucm2/codecs/nau8824/EnableSeq.conf
@@ -7,8 +7,8 @@ EnableSequence [
 	cset "name='DMIC2 Enable Switch' off"
 	cset "name='DMIC3 Enable Switch' off"
 	cset "name='DMIC4 Enable Switch' off"
-	cset "name='MIC1 Volume' 10"
-	cset "name='MIC2 Volume' 10"
+	cset "name='MIC1 Volume' 14"
+	cset "name='MIC2 Volume' 14"
 	# Button Configuration
 	cset "name='THD for key media' 10"
 	cset "name='THD for key voice command' 16"
-- 
2.26.2

